.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-python
+
+name: Run Acceptance and Unit tests
```

Comment:
This is very interesting! And has been on my wish list to include for some time.